app/app_logging.py

```diff
@@ -20,6 +20,13 @@ class CustomJsonFormatter(jsonlogger.JsonFormatter):
     lib_dir = os.path.normpath(os.path.split(sys.executable)[0] + '/..')
     cw_dir = os.getcwd()
 
+    def formatTime(self, record, datefmt=None):
+        date_format = '%Y-%m-%dT%H:%M:%S'
+        msec_format = '%s.%03d'
+        ct = self.converter(record.created)
+        t = time.strftime(date_format, ct)
+        return msec_format % (t, record.msecs)
+
```

Comment:
Can this be commented as being an override function, and what it is doing - it's not clear at all from the code

Reply:
done

Reply:
Since I provided this code to Akriti, here are the additional comments requested:
```
  def formatTime(self, record, datefmt=None):
        """
        Return the creation time of the specified LogRecord as formatted text, overriding the
        superclass version to ensure that millisecond granularity are recorded.

        The superclass version only has second granularity since that is all time.strftime provides.
        This method combines the millisecond parts of the time, with the formatted time (to second granularity)
        to provide a millisecond formatted result.
        """
        date_format = '%Y-%m-%dT%H:%M:%S'
        msec_format = '%s.%03d'
        ct = self.converter(record.created)
        t = time.strftime(date_format, ct)
        return msec_format % (t, record.msecs)
```

Reply:
added comment provided by Rob

Reply:
Happy with Rob's updated commenting